src/test/java/org/openrewrite/java/testing/mockito/ReplacePowerMockitoIntegrationTest.java

```diff
@@ -0,0 +1,88 @@
+package org.openrewrite.java.testing.mockito;
+
+import static org.openrewrite.java.Assertions.java;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.config.Environment;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+class ReplacePowerMockitoIntegrationTest implements RewriteTest {
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.parser(JavaParser.fromJavaVersion()
+            .classpathFromResources(new InMemoryExecutionContext(), "mockito-all-1.10.19", "junit-4.13.2", "hamcrest-2.2", "junit-jupiter-api-5.9.2"))
```

Comment:
Would this not also need `powermock-core` and/or `powermock-api-mockito`?

Reply:
Yeah, I would be a bit surprised that the tests pass without entries for those